code/go/0chain.net/chaincore/state/mint.go

```diff
@@ -9,14 +9,13 @@ import (
 
 var ErrInvalidMint = common.NewError("invalid_mint", "invalid minter")
 
-//Transfer - a data structure to hold state transfer from one client to another
 type Mint struct {
 	Minter     datastore.Key `json:"minter"`
 	ToClientID datastore.Key `json:"to"`
 	Amount     Balance       `json:"amount"`
 }
 
-//NewTransfer - create a new transfer
+// NewMint NewTransfer - create a new transfer
```

Comment:
New mint does not create a transfer, it mints tokens.

Reply:
Agree, not my comment. I fixed the IDE warning, I shouldn't change the code not related to PR